soul-metrics-prometheus JmxScraperTest bug

## Describe the bug
## JmxScraperTest
- testJmxScraperWithNonEmptyJmxUrl test error
- testJmxScraperWithNonEmptyJmxUrlThenReceivedByReceiverDefaultExport test error
## Environment

- Soul version(s): [e.g. lastest]

## Steps to reproduce
run test


## Expected behavior

## Debug logs
<!--
Add your debug logs here.
-->

<details>
<summary>Click to expand</summary>

```
java.rmi.ConnectIOException: error during JRMP connection establishment; nested exception is: 
	java.net.SocketTimeoutException: Read timed out

	at sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:307)
	at sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:202)
	at sun.rmi.server.UnicastRef.invoke(UnicastRef.java:129)
	at javax.management.remote.rmi.RMIServerImpl_Stub.newClient(Unknown Source)
	at javax.management.remote.rmi.RMIConnector.getConnection(RMIConnector.java:2430)
	at javax.management.remote.rmi.RMIConnector.connect(RMIConnector.java:308)
	at javax.management.remote.JMXConnectorFactory.connect(JMXConnectorFactory.java:270)
	at org.dromara.soul.metrics.prometheus.impl.collector.JmxScraper.doScrape(JmxScraper.java:137)
	at org.dromara.soul.metrics.prometheus.impl.collector.JmxScraperTest.testJmxScraperWithNonEmptyJmxUrlThenReceivedByReceiverDefaultExport(JmxScraperTest.java:187)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:220)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:53)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:171)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readByte(DataInputStream.java:265)
	at sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:246)
	... 32 more
```
</details>

--- a/soul-metrics/soul-metrics-prometheus/src/test/java/org/dromara/soul/metrics/prometheus/impl/collector/BuildInfoCollectorTest.java
+++ b/soul-metrics/soul-metrics-prometheus/src/test/java/org/dromara/soul/metrics/prometheus/impl/collector/BuildInfoCollectorTest.java
@@ -17,16 +17,10 @@
 
 package org.dromara.soul.metrics.prometheus.impl.collector;
 
-import io.prometheus.client.Collector.MetricFamilySamples;
-import io.prometheus.client.GaugeMetricFamily;
+import org.junit.Before;
 import org.junit.Test;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertNotNull;
 
 /**
  * The Test Case For BuildInfoCollector.
@@ -35,20 +29,15 @@ import static org.junit.Assert.assertThat;
  **/
 public final class BuildInfoCollectorTest {
 
+    private BuildInfoCollector buildInfoCollector;
+
+    @Before
+    public void setUp() {
+        buildInfoCollector = new BuildInfoCollector();
+    }
+
     @Test
-    public void collect() {
-        List<MetricFamilySamples> mfs = new ArrayList<>();
-        GaugeMetricFamily artifactInfo = new GaugeMetricFamily(
-                "jmx_exporter_build_info",
-                "A metric with a constant '1' value labeled with the version of the JMX exporter.",
-                Arrays.asList("version", "name"));
-        Package pkg = this.getClass().getPackage();
-        String version = pkg.getImplementationVersion();
-        String name = pkg.getImplementationTitle();
-        artifactInfo.addMetric(Arrays.asList(version != null ? version : "unknown", name != null ? name : "unknown"), 1L);
-        mfs.add(artifactInfo);
-        assertThat(mfs.toString(), is("[Name: jmx_exporter_build_info Type: GAUGE Help: "
-                + "A metric with a constant '1' value labeled with the version of the JMX exporter. Samples: "
-                + "[Name: jmx_exporter_build_info LabelNames: [version, name] labelValues: [unknown, unknown] Value: 1.0 TimestampMs: null]]"));
+    public void testBuildInfoCollector() {
+        assertNotNull(buildInfoCollector.collect());
     }
 }

--- a/soul-metrics/soul-metrics-prometheus/src/test/java/org/dromara/soul/metrics/prometheus/impl/collector/JmxCollectorTest.java
+++ b/soul-metrics/soul-metrics-prometheus/src/test/java/org/dromara/soul/metrics/prometheus/impl/collector/JmxCollectorTest.java
@@ -27,10 +27,10 @@ import java.util.Collections;
 import java.util.regex.Pattern;
 
 import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertFalse;
 
 /**
  * The type Jmx collector test.

